troupon/templates/snippet_sidebar_item.html

```diff
@@ -0,0 +1,9 @@
+{% if request.path == url or active_tab|slugify == name|slugify %}
```

Comment:
**Code Quality: Nitpick**

The spacing here is not similar to what you've been using prior. Stick to 2 spaces.